teamB/jungJaeUk/app.js

```diff
@@ -6,50 +6,19 @@ const morgan = require('morgan');
 const dotenv = require('dotenv');
 dotenv.config();
```

Comment:
- dotenv의 경우 아래와 같이 수정한 후 최상단에 작성해주세요 !
```suggestion
require('dotenv').dotenv.config();
```